js/bookmarker.js

```diff
@@ -2,9 +2,9 @@
  * Content script to inject the dynamic bookmarking JS directly into the page.
  */
 (function() {
-  function chromeOrBrowser() {
-    return this.browser || chrome;
-  }
+  // function chromeOrBrowser() {
```

Comment:
Can only use `chrome` now. When uploading the manifest it complains if this wasn't dealt with.